test/composableStablePool.spec.ts

```diff
@@ -1,19 +1,34 @@
 // TS_NODE_PROJECT='tsconfig.testing.json' npx mocha -r ts-node/register test/composableStablePool.spec.ts
```

Comment:
I think these are just testing same as the queryJoin/Exit against hard fork? We could remove this and new .json files.

Reply:
You're right! I'll remove them 👍

Reply:
Think you still need to remove json test files that were previously added?